support/ebpf/types.h

```diff
@@ -331,6 +331,17 @@ typedef enum TracePrograms {
   NUM_TRACER_PROGS,
 } TracePrograms;
 
+// TraceOrigin describes the source of the trace. This enables
+// origin specific handling of traces in user space.
+typedef enum TraceOrigin {
+  TRACE_UNKNOWN,
+  TRACE_SAMPLING,
+  TRACE_OFF_CPU,
+} TraceOrigin;
+
+// OFF_CPU_THRESHOLD_MAX defines the maximum threshold.
+#define OFF_CPU_THRESHOLD_MAX 1000
```

Comment:
This should probably be 100 which is intuitive percentage-wise and also consistent with the probabilistic profiling setting:

```suggestion
#define OFF_CPU_THRESHOLD_MAX 100
```

Reply:
1000 is chosen over 100 to provide a more fine grained possibility to configure the off-CPU threshold. On a single core system, using regular percentages makes sense, I think. But the more cores are in use, the more data will be generated. And so I think it makes sense to provide the option to reduce the storage and do off-CPU profiling only for a smaller amount than 1% of all scheduler task switches.
Let me know, if 1% of all scheduler task switches is the minimum that should be allowed to configure.

Reply:
Need to check if hitting on less than 1% of all scheduler task switches provides value given the current sampling strategy by running some test workloads.

Reply:
Setting the minimum threshold to 1% results in 1% or less CPU usage on my system so I don't think we need to go lower as 1-100 is more intuitive, but not a blocker.